tokio-console/src/config.rs

```diff
@@ -28,7 +32,7 @@ pub struct Config {
     #[clap(long = "log", env = "RUST_LOG", default_value = "off")]
     pub(crate) env_filter: tracing_subscriber::EnvFilter,
 
-    #[clap(flatten)]
+    #[clap(skip)]
```

Comment:
The reason I skipped it:
- I don't know how to use default values from config files.
- If the parameter is enabled, I should implement that:
    - If command-line options are given, then need to use them instead of the config file's value.